packages/lutin/lutin.2.56/opam

```diff
@@ -16,7 +16,7 @@ remove: [
   ["ocaml" "%{etc}%/Lutin/setup.ml" "-C" "%{etc}%/Lutin" "-uninstall"]
 ]
 depends: [
-  "ocaml" {>= "4.02" & < "4.12"}
+  "ocaml" {>= "4.02" & < "4.12" & != "4.03"}
```

Comment:
I disagree with this change. The failure is due to some incorrect lower bound in one of the dependencies. The package works fine on 4.03 in general

Reply:
```suggestion
  "ocaml" {>= "4.02" & < "4.12"}
```

Reply:
ok